src/VAC/VectorAnimationComplex/KeyFace.h

```diff
@@ -87,6 +87,18 @@ class Q_VPAINT_EXPORT KeyFace: public KeyCell, public FaceCell
 
     void updateInfill();
 
+    QColor infillColor() const;
+    void setIffillColor(const QColor& c);
+
+    QColor infillHighlightedColor() const;
+    void setInfillHighlightedColor(const QColor& c);
+
+    QColor infillSelectedColor() const;
```

Comment:
[[nodiscard]] on simple getters doesn't hurt, not critical at all though.

Reply:
done